src/main/java/io/nats/client/support/JsonValueUtils.java

```diff
@@ -319,68 +321,28 @@ public MapBuilder put(String s, Object o) {
             return this;
         }
 
-        public MapBuilder put(String s, Map<String, String> stringMap) {
+        public MapBuilder put(String key, Map<String, String> stringMap) {
             if (stringMap != null) {
                 MapBuilder mb = new MapBuilder();
-                for (String key : stringMap.keySet()) {
-                    mb.put(key, stringMap.get(key));
+                for (Map.Entry<String, String> entry : stringMap.entrySet()) {
+                    mb.put(entry.getKey(), entry.getValue());
                 }
-                jv.map.put(s, mb.jv);
-                jv.mapOrder.add(s);
+                jv.map.put(key, mb.jv);
+                jv.mapOrder.add(key);
             }
             return this;
         }
 
         @Override
-        @NonNull
-        public String toJson() {
+        public @NonNull String toJson() {
```

Comment:
I considered both ways and I prefer these where they where, on a separate line before the method.